apps/generate.nix

```diff
@@ -144,7 +144,7 @@ let
             || die "Failed to add generated secret to git"
         fi
       else
-        echo "[1;90m    Skipping[m [90m[already exists] "${escapeShellArg contextSecret.sourceFile}" ("${concatStringsSep "', '" (map escapeShellArg contextSecret.defs)}")[m"
+        echo "[1;90m    Skipping[m [90m[already exists] ${escapeShellArg contextSecret.sourceFile}: ${concatStringsSep ", " (map escapeShellArg contextSecret.defs)}[m"
```

Comment:
It's been a minute since I've made these changes, but my vague recollection was that I'd stumbled across this error, but the quoting here caused the error reporting itself to generate an error.  I'm happy to split this into a separate commit or even PR if desired.

Reply:
This quote is definitely needed in some form, since escapeShellArg expects to not be in a quoted context so we need to end the `"` context before using escapeShellArg, otherwise we will have `'` characters included in the output and maybe some other edge cases.

I don't see how the old version can even break, do you have an example?

Reply:
I don't have one on hand unfortunately.  I'll back it out and can put it back in if I run into it again, with an example / test to prove it out.

Reply:
Apologies this took so long.  I've reverted the change.  If I bump across the issue I thought I observed again, I'll make sure to have a better reproduction documented.